daemon-set/scan_data.yml

```diff
@@ -0,0 +1,20 @@
+apiVersion: extensions/v1beta1
+kind: DaemonSet
+metadata:
+  name: scan-data
+spec:
+  selector:
+    matchLabels:
+      name: scan-data
+  template:
+    metadata:
+      labels:
+        name: scan-data
+    spec:
+      nodeSelector:
+        zone: default
+      containers:
+      - image: registry.centos.org/navidshaikh/ccp-openshift-scan
```

Comment:
> Can we move this to pipeline-images instead of navidshaikh?

Ack!

> Also can we add the files required to build this image to this repo? Maybe after this PR is merged.

Are you mentioning the scan module scripts? if yes, they are part of PR #12 .